platform/linux-generic/odp_packet.c

```diff
@@ -2843,3 +2843,10 @@ odp_proto_l4_type_t odp_packet_l4_type(odp_packet_t pkt)
 
 	return ODP_PROTO_L4_TYPE_NONE;
 }
+
+uint64_t odp_packet_mark_id(odp_packet_t pkt)
+{
+	(void)pkt;
```

Comment:
ODP seems to typically use ODP_UNUSED attribute in the parameter list instead of this.

Reply:
I think both ways are used, and I'm OK with either way. Personally, I prefer cast to void as it's standard C language (attribute supports is compiler specific).

Reply:
OK. That case, I prefer to keep (void).